modin/pandas/test/utils.py

```diff
@@ -692,6 +692,14 @@ def generate_multiindex(index):
     return df1, df2
 
 
+def generate_multiindex(cols_number):
+    arrays = [
+        np.random.choice(["bar", "baz", "foo", "qux"], cols_number),
+        np.random.choice(["one", "two"], cols_number),
+    ]
+    return pd.MultiIndex.from_tuples(list(zip(*arrays)), names=["first", "second"])
```

Comment:
I think it's better to use `random_state` from `tests/utils.py` with established seed to make tests reproducible
https://github.com/modin-project/modin/blob/82849bb9ccfb0cde899a02386f6f3b9bc336a333/modin/pandas/test/utils.py#L27

Reply:
Done